src/components/filter/FilterBoilerplate.vue

```diff
@@ -34,7 +33,8 @@
             v-model="query"
             class="filter__items__search"
             :placeholder="$t('search.searchIn') + ' ' + $t('filter.' + filter.name) + '...'"
-            :rounded="false"
+            rounded="false"
+            dark="true"
```

Comment:
Boolean prop doesn't need a value when they have a `false` as default value.

In that case you also pass "false" (as a string) to the rounded prop which expects a boolean.

So the properties should be written:

```
:rounded="false"
dark
```

Reply:
Note that it's not very intuitive to have positive default value for `autofocus` and `rounded` on this component. We might refactor their default values to `false` and update the place where this component is used.